source/features/tab-size.css

```
@@ -1,41 +1,15 @@
 /* Make tab-indented code more readable on GitHub and Gist */
 
-* {
-	tab-size: var(--tab-size) !important;
-}
-
-[data-tab-size='2'] {
-	--tab-size: 2;
-}
-
 :root, /* Changes the browser default */
-.comment-body [data-tab-size='8'], /* User setting is ignored in comments #4833 */
-[data-tab-size='4'] {
-	--tab-size: 4;
-}
-
-[data-tab-size='8'] {
-	--tab-size: 8;
-}
-
-[data-tab-size='10'] {
-	--tab-size: 10;
+.comment-body .tab-size[data-tab-size='8'] /* User setting is ignored in comments #4833 */ {
+	tab-size: 4;
```

The solution here is to preserve that `tab-size: var(--tab-size)` to never have these inconsistencies:

```suggestion
	tab-size: var(--tab-size);
```

And also keep this only in the root to allow a global override:

```css
:root {
	--tab-size: 4;
}
```

---

Also you can follow your own review here 😁 

https://github.com/refined-github/refined-github/pull/4833#discussion_r733884651